altiProfil/classes/altiProfil.listener.php

```diff
@@ -95,6 +96,12 @@ function ongetMapAdditions ($event)
                     $js[] = $bp.'assets/js/dataviz/plotly-latest.min.js';
                     $js[] = $bp.'assets/js/dataviz/dataviz.js';
                 }
+                if (file_exists(jApp::wwwPath('js/dataviz/plotly-locale-'.$locale.'-latest.js'))) {
```

Comment:
this test seems useless, the plotly locale files are in `assets/js/dataviz/` directory as in your code below
Any reason you check the `js` dir?